README.md

```diff
@@ -27,103 +32,103 @@ Then, build the project with:
 make build
 ```
 
-### Generating an example
+**voodoo** is run in three successives steps detailed below.
 
-To create an example, run:
+## :seedling: 1. Prepare the packages: voodoo-prep
 
-```bash
-make example
+To prepare the packages just run:
+
+```sh
+voodoo-prep [-u <UNIVERSES>]
 ```
 
-This will create a sample website with the documentation of `ocaml-base-compiler` in `_generated/output/p`.
+Where `<UNIVERSES>` is a comma-separated list of packages and universes `pkg1:unv1,pkg2:unv2,...`.
 
-Here are some commands that will be executed when running the above:
+Each package installed in the current switch will be prepared (prepped) for the compiling step.
```

Comment:
When you don't provide a universe explicitly, each package installed in the current switch will be prepared for the compiling step.

(This is only for local use as I understand, since in the context of ocaml-docs-ci, the universe generation happens on ocaml-docs-ci.)